core/src/components/input/input.ionic.scss

```diff
@@ -1,26 +1,127 @@
 @use "../../themes/ionic/ionic.globals.scss" as globals;
 @use "./input.common";
-@forward "./input.ionic.outline.scss";
-
+@forward "./input.ionic.solid.scss";
 // Ionic Input
 // --------------------------------------------------
 
 :host {
   --color: #{globals.$ion-primitives-neutral-1200};
   --border-width: #{globals.$ion-border-size-025};
   --border-color: #{globals.$ion-primitives-neutral-500};
+  --border-radius: #{globals.$ion-border-radius-100};
   --highlight-color-valid: #{globals.$ion-semantics-success-900};
-  --highlight-color-invalid: #{globals.$ion-semantics-danger-800};
+  --highlight-color-invalid: #{globals.$ion-border-danger-default};
+  --padding-start: #{globals.$ion-space-300};
+  --padding-end: #{globals.$ion-space-300};
```

Comment:
This seems to only apply for the `small` size - can we move this to `.input-size-small`?